### ISSUE TITLE
Error : no release for just matching version specifier 0.1

### ISSUE THREAD

**Issue by alelouis** [extractions/setup-just#4]
Hi,

I tired to use the setup-just action in my own workflow. Unfortunatly I'm confronted to this error message (tested for version 0.8, 0.10 and latest (without version)).

![image](https://user-images.githubusercontent.com/6841652/131569536-b1dcc1f1-3766-4644-b4ae-7b808bb5fb8f.png)

Do any of you encounter this issue ? 

Example of usage in the workflow:

```yml
build:
    runs-on: ubuntu-latest
    needs: test
    steps:
      - uses: actions/checkout@v2
      - uses: extractions/setup-just@v1
        with:
          just-version: 0.10
      - uses: actions/setup-node@v1
        with:
          node-version: '14'
      - uses: docker/login-action@v1 
        with:
          username: ${{ secrets.DOCKERHUB_USERNAME }}
          password: ${{ secrets.DOCKERHUB_TOKEN }}
      - uses: docker/setup-qemu-action@v1
      - uses: docker/setup-buildx-action@v1
        id: buildx
        with:
          install: true

```

**Comment by rossmacarthur** [comment]
Hi @alelouis, I think this is because in YAML 0.10 means a float. Can you try

```yaml
      - uses: extractions/setup-just@v1
        with:
          just-version: '0.10'
```

I have fixed the README in bd335a2

**Comment by alelouis** [comment]
Silly me. Thanks a lot @rossmacarthur.
